validator/validators/stress/stress_validator.go

```diff
@@ -28,39 +29,69 @@ var (
 	metricPluginBoundValue = MetricPluginBoundValue{
 		"1000": {
 			"statsd": {
-				"procstat_cpu_usage":   float64(19),
-				"procstat_memory_rss":  float64(66500000),
+				"procstat_cpu_usage":   float64(25),
+				"procstat_memory_rss":  float64(82000000),
 				"procstat_memory_swap": float64(0),
 				"procstat_memory_vms":  float64(818000000),
-				"procstat_memory_data": float64(95000000),
-				"procstat_num_fds":     float64(9),
-				"net_bytes_sent":       float64(100000),
-				"net_packets_sent":     float64(100),
+				"procstat_memory_data": float64(83000000),
+				"procstat_num_fds":     float64(11),
+				"net_bytes_sent":       float64(105000),
+				"net_packets_sent":     float64(105),
+			},
+			"logs": {
```

Comment:
Adding the bound value for logs which will served as [a upper bound value for CWA when monitor logs which CWA cannot cross](https://github.com/aws/amazon-cloudwatch-agent-test/pull/155/files#diff-47c87373e751dd9fd5ce504e44b320765c8b84d6cde524a4e8a32cfa34674165R231-R236).